config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -272,7 +263,7 @@ periodics:
     preset-bazel-scratch-dir: "true"
     preset-cloudflare-credentials: "true"
     preset-venafi-tpp-credentials: "true"
-    preset-venafi-cloud-credentials: "true"
+    preset-k8s-v19-plus-env: "true"
```

Comment:
Bit of a nit, but it took me quite a while to actually work out what this preset does due to the name of it being a description of the environment rather than a description of what it does (e.g. `preset-enable-k8s-csr-api-tests`). Could the preset be renamed?

Also, it may be better to _invert_ this so that eventually it is not required once k8s <1.19 is no longer in use? So.. `preset-skip-k8s-csr-tests` added to each e2e job that is LESS THAN kubernetes 1.19?

Reply:
Ah, agreed for _gateway_ - but I was referring to the `csr` (i.e. kubernetes CertificateSigningRequest) feature gates here, rather than gateway... perhaps I got mixed up :)

Reply:
Ahh no that Gateway bit didn't make any sense- the preset is used to enable/disable both feature gates https://github.com/jetstack/testing/commit/875eb40cef7f9b08f1b74b1f78da0780f5f7e8ec#diff-ce3d91fd77957f12a21c34a9d5b2805cf9c1b0fd5be4cc57a6372e9bc7fe500eL102 so we can't really split I think, IMO it's still better to be explicit that those both are enabled by default for as long as they are feature gates.